packages/php-wasm/logger/src/lib/logger.ts

```diff
@@ -248,14 +292,40 @@ export function collectPhpLogs(
 }
 
 /**
- * Add a listener for the fatal Playground errors.
- * These errors include Playground errors like Asyncify errors. PHP errors won't trigger this event.
+ * Collect worker metrics.
+ *
+ * @param worker The service worker
+ */
+export function collectWorkerMetrics(worker: ServiceWorkerGlobalScope) {
+	worker.addEventListener('activate', (event) => {
+		event.waitUntil(worker.clients.claim());
```

Comment:
What does it do and why is it needed for logging?

Reply:
This will trigger the `controllerchange` event in other clients and update the count. I'm adding a comment to the code.

Reply:
[c37a47e](https://github.com/WordPress/wordpress-playground/pull/1253/commits/c37a47e593b8203df5212ccb109e95120a7df895)

Reply:
I remember having the `.claim()` call in the service worker earlier on and then having to remove it because it interfered with loading / updating Playground somehow. The setup is different now so it might be fine, but I will still ask – what are the potential unintended consequences of introducing that call? Especially when a few tabs are open, a new service worker is enqueued etc.

Reply:
> When a service worker is initially registered, pages won't use it until they next load. The claim() method causes those pages to be controlled immediately. Be aware that this results in your service worker controlling pages that loaded regularly over the network, or possibly via a different service worker. 

[This seems related](https://developer.mozilla.org/en-US/docs/Web/API/Clients/claim)

Reply:
Everything seems to work. If I have a few tabs open, they can make requests, adding more tabs doesn't create any issues, and if I make changes in the code, the browser updates the SW on reload.